crates/sui-benchmark/src/bin/stress.rs

```diff
@@ -404,8 +410,11 @@ async fn main() -> Result<()> {
                     stat_collection_interval,
                     shared_counter,
                     transfer_object,
+                    shared_counter_hotness_factor,
                     ..
                 } => {
+                    let shared_counter_ratio = 1.0
+                        - (std::cmp::min(shared_counter_hotness_factor as u32, 100) as f32 / 100.0);
```

Comment:
it probably doesn't matter but you may as well use f64 here.

Reply:
sure thing, let me add another comment explaining it better